com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDEditorUtils.cs

```diff
@@ -276,5 +276,11 @@ internal static void HandlePrefixLabelWithIndent(Rect totalPosition, Rect labelP
             labelPosition.x += EditorGUI.indentLevel * 15;
             EditorGUI.HandlePrefixLabel(totalPosition, labelPosition, label);
         }
+
+        // IsPreset is an internal API - lets reuse the usable part of this function
+        static internal bool IsPresetEditor(UnityEditor.Editor editor)
+        {
+            return (int)((editor.target as Component).gameObject.hideFlags) == 93;
```

Comment:
Any case where editor or editor.target can be null?

Reply:
not as far as i know, I am passing "this" every time